[2025-05-18T15:31:55.087+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_pipeline.start_services manual__2025-05-17T18:33:49+00:00 [queued]>
[2025-05-18T15:31:55.119+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_pipeline.start_services manual__2025-05-17T18:33:49+00:00 [queued]>
[2025-05-18T15:31:55.121+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-05-18T15:31:55.187+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): start_services> on 2025-05-17 18:33:49+00:00
[2025-05-18T15:31:55.198+0000] {standard_task_runner.py:57} INFO - Started process 476 to run task
[2025-05-18T15:31:55.212+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'data_pipeline', 'start_services', 'manual__2025-05-17T18:33:49+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/data_pipeline.py', '--cfg-path', '/tmp/tmp5xqdy799']
[2025-05-18T15:31:55.227+0000] {standard_task_runner.py:85} INFO - Job 3: Subtask start_services
[2025-05-18T15:31:55.423+0000] {task_command.py:416} INFO - Running <TaskInstance: data_pipeline.start_services manual__2025-05-17T18:33:49+00:00 [running]> on host 372a9807070e
[2025-05-18T15:31:55.764+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_pipeline' AIRFLOW_CTX_TASK_ID='start_services' AIRFLOW_CTX_EXECUTION_DATE='2025-05-17T18:33:49+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-17T18:33:49+00:00'
[2025-05-18T15:31:55.772+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-05-18T15:31:55.776+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo "Starting services..."']
[2025-05-18T15:31:55.800+0000] {subprocess.py:86} INFO - Output:
[2025-05-18T15:31:55.805+0000] {subprocess.py:93} INFO - Starting services...
[2025-05-18T15:31:55.807+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-05-18T15:31:55.871+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=data_pipeline, task_id=start_services, execution_date=20250517T183349, start_date=20250518T153155, end_date=20250518T153155
[2025-05-18T15:31:55.949+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-05-18T15:31:56.032+0000] {taskinstance.py:2778} INFO - 2 downstream tasks scheduled from follow-on schedule check
